SPECS/kernel.spec

```
@@ -1044,6 +1045,9 @@ fi
 %{?_cov_results_package}
 
 %changelog
+* Sun Apr 6 2025 Anthoine Bourgeois <anthoine.bourgeois@vates.tech> - 4.19.19-8.0.38.2
```

It is customary to ensure the day number is on two characters in changelog entries.

```suggestion
* Sun Apr 06 2025 Anthoine Bourgeois <anthoine.bourgeois@vates.tech> - 4.19.19-8.0.38.2
```